src/dbus_mpris.rs

```diff
@@ -562,12 +554,52 @@ async fn create_dbus_server(
         }
     });
 
+    let spotifyd_ctrls_interface: IfaceToken<()> = cr.register("io.github.spotifyd.Controls", |b| {
+        let local_spirc = spirc.clone();
+        b.method("VolumeUp", (), (), move |_, _, (): ()| {
+            local_spirc.volume_up();
+            Ok(())
+        });
+        let local_spirc = spirc.clone();
+        b.method("VolumeDown", (), (), move |_, _, (): ()| {
+            local_spirc.volume_down();
+            Ok(())
+        });
+
+        let mv_device_name = device_name.clone();
+        let sp_client = Arc::clone(&spotify_api_client);
+        b.method("TransferPlayback", (), (), move |_, _, (): ()| {
+            let device_id = get_device_id(&sp_client, &mv_device_name, false);
+            if let Some(device_id) = device_id {
+                info!("Transferring playback to device {}", device_id);
+                match sp_client.transfer_playback(&device_id, Some(true)) {
+                    Ok(_) =>Ok(()),
+                    Err(err) => {
+                        let e = format!("RSpotify error: {}", err);
+                        error!("{}", e);
+                        Err(MethodErr::failed(&e))
+                    },
+                }
+            } else {
+                let msg = format!("Could not find device with name {}", mv_device_name);
+                warn!("{}", msg);
```

Comment:
```suggestion
                warn!("TransferPlayback: {}", msg);
```

Maybe, we could add some context here?